TA_dataset/bin/dataset_api.py

```diff
@@ -5,7 +5,7 @@
 
 # adjust paths to make the Splunk app working
 import import_declare_test  # noqa: F401
-from dataset_common import logger, normalize_time
+from dataset_common import get_tenant_related_payload, logger, normalize_time
```

Comment:
This is not used here, after the refactoring

Reply:
done